### PR TITLE
Create route for global view

### PR DESCRIPTION
**Pull request by nmuruli**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -9,8 +9,12 @@ app.use(bodyParser.urlencoded({ extended: true }));
 app.use(bodyParser.json());
 app.use(cors());
 
-//importing patient routes
+//importing global view routes
```
Comment (jinzihuang) [inline server.js]: This should be global route

File: api/routes/globalRoutes.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = function(app) {
+
+    // Use Global View controller to pull data from the model (MySQL DB), 
+    // convert it to JSON and return it to the route.
+
+    var patients = require('../controllers/globalController');
```
Comment (jinzihuang) [inline api/routes/globalRoutes.js]: better to use global instead of patients

File: api/routes/globalRoutes.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+    // API for Global View ROW 2
+    app.route('/global')
+    .get(patients.global);
```
Comment (jinzihuang) [inline api/routes/globalRoutes.js]: same here

File: server.js (hunk 1 above)
Comment (nmuruli) [reply server.js]: Done

File: api/routes/globalRoutes.js (hunk 2 above)
Comment (nmuruli) [reply api/routes/globalRoutes.js]: Done

File: api/routes/globalRoutes.js (hunk 3 above)
Comment (nmuruli) [reply api/routes/globalRoutes.js]: Done

**Pull request merged**